data-protocols/ids/ids-api-multipart-endpoint-v1/src/main/java/org/eclipse/dataspaceconnector/ids/api/multipart/handler/DescriptionHandler.java

```diff
@@ -47,15 +47,14 @@
     private final ResourceDescriptionRequestHandler resourceDescriptionRequestHandler;
     private final ConnectorDescriptionRequestHandler connectorDescriptionRequestHandler;
 
-    public DescriptionHandler(
-            @NotNull Monitor monitor,
-            @NotNull String connectorId,
-            @NotNull TransformerRegistry transformerRegistry,
-            @NotNull ArtifactDescriptionRequestHandler artifactDescriptionRequestHandler,
-            @NotNull DataCatalogDescriptionRequestHandler dataCatalogDescriptionRequestHandler,
-            @NotNull RepresentationDescriptionRequestHandler representationDescriptionRequestHandler,
-            @NotNull ResourceDescriptionRequestHandler resourceDescriptionRequestHandler,
-            @NotNull ConnectorDescriptionRequestHandler connectorDescriptionRequestHandler) {
+    public DescriptionHandler(@NotNull Monitor monitor,
```

Comment:
Minor nit: If this is just formatting, is it possible to avoid this since it makes it difficult to see what the actual changes are?